demos/html5/ckeditor5/package.json

```diff
@@ -24,9 +24,7 @@
     "@ckeditor/ckeditor5-essentials": ">=22.0.0",
     "@ckeditor/ckeditor5-paragraph": ">=22.0.0",
     "@ckeditor/ckeditor5-theme-lark": ">=22.0.0",
-    "@wiris/mathtype-ckeditor5": "^7.24.0",
-    "postcss-loader": "^3.0.0",
-    "webpack-dev-server": "^3.10.3"
```

Comment:
Why delete webpack dev-server from dependencies?

Reply:
I didn't. I moved it, along with 'postcss-loader' to the "devDependencies" group.
The "webpack-dev-server" was already indicated, so in fact, it was duplicated.